ui/src/lib/features/api-auth/store.ts

```diff
@@ -3,4 +3,4 @@
 import { writable } from 'svelte/store'
 
 export const authenticated = writable(false)
-export const apiAuthEnabled = writable(false)
+export const apiAuthEnabled = writable(<null | boolean>null)
```

Comment:
i've never seen the typing like this, but the type check isn't failing so.... is there a particular reason for not doing 
```suggestion
export const apiAuthEnabled = writable<null | boolean>(null)
```